### ISSUE TITLE
Interactive highlight for pickup coil changes as light rays change.

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/faradays-electromagnetic-lab#189]
Related to https://github.com/phetsims/faradays-electromagnetic-lab/issues/188 ...

The interactive highlight for the pickup coil changes as the light rays change.  Apparently the interactive highlight is based on the full bounds of the interactive Node, not just the interactive parts.

This probably does not block publication, but is really annoying when using the sim. And it seems like a general problem that should be addressed.

**Comment by pixelzoom** [comment]
In 6/12/24 planning meeting...

@kathy-phet suggested adding an option to exclude non-interactive Nodes from the bounds used to create the interactive highlight. @jonathanolson and @jessegreenberg were not sure how difficult/feasible this is.

@marlitas suggested having the sim take control of setting interactiveHighlight, as was done in MSAB.

**Comment by pixelzoom** [comment]
> @kathy-phet suggested adding an option to exclude non-interactive Nodes from the bounds used to create the interactive highlight.

@jessegreenberg can you comment of the feasibility/complexity of adding such an option?

**Comment by pixelzoom** [comment]
@arouinfar and I worked on this, and decided that it was undesirable for the user to drag the pickup coil by the light bulb or voltmeter.   So we are now explicilty setting focusHighlight (and therefore interactiveHighlight) by computing the bounding rectangle of the coil.

So we do not need the option suggested by @kathy-phet.  @jessegreenberg if you think that option is worth investigating, feel free to create a scenery issues.

Closing.
